ssv/sync_committee.go

```diff
@@ -149,7 +149,7 @@ func (r *SyncCommitteeRunner) expectedPostConsensusRootsAndDomain() ([]ssz.HashR
 func (r *SyncCommitteeRunner) executeDuty(duty *types.Duty) error {
 	// TODO - waitOneThirdOrValidBlock
 
-	root, err := r.GetBeaconNode().GetSyncMessageBlockRoot()
+	root, err := r.GetBeaconNode().GetSyncMessageBlockRoot(duty.Slot)
```

Comment:
why does it require a slot? Prysm doesn't pass a slot

Reply:
go-client repos request it as it support all clients. 
also for this request it need it. you used functions from prysm but we use api calls